js/main.js

```diff
@@ -15,15 +15,77 @@ $photoUrl.addEventListener('input', handlePhotoUrl);
 function handleForm(event) {
   event.preventDefault();
   var inputObject = {};
-
   inputObject.title = event.target.title.value;
   inputObject.photo = event.target.photo.value;
   inputObject.notes = event.target.notes.value;
   data.nextEntryId++;
   inputObject.entryId = data.nextEntryId;
   data.entries.unshift(inputObject);
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  var newEntry = renderEntry(inputObject);
+  var ulList = document.querySelector('ul');
+  ulList.prepend(newEntry);
   $form.reset();
+  switchingViews('entry-form');
+
 }
 
 $form.addEventListener('submit', handleForm);
+
+function renderEntry(entry) {
+  var liElement = document.createElement('li');
+
+  var mainRow = document.createElement('div');
+  mainRow.setAttribute('class', 'row');
+  liElement.appendChild(mainRow);
+
+  var entryImage = document.createElement('img');
+  entryImage.setAttribute('class', 'column-full column-half');
+  entryImage.setAttribute('src', entry.photo);
+  mainRow.appendChild(entryImage);
+
+  var columnHalf = document.createElement('div');
+  columnHalf.setAttribute('class', 'column-half');
+  mainRow.appendChild(columnHalf);
+
+  var entryTitle = document.createElement('h2');
+  entryTitle.textContent = entry.title;
+  columnHalf.appendChild(entryTitle);
+
+  var note = document.createElement('p');
+  note.textContent = entry.notes;
+  columnHalf.appendChild(note);
+
+  return liElement;
+}
+
+var list = document.querySelector('#list');
+
+window.addEventListener('DOMContentLoaded', event => {
+  for (var i = 0; i < data.entries.length; i++) {
+    var result = renderEntry(data.entries[i]);
+    list.appendChild(result);
+    switchingViews(data.view);
```

Comment:
You can do this above or below the loop, it doesn't need to be done on each iteration.